adobe_vipm/flows/sync.py

```diff
@@ -767,6 +767,9 @@ def sync_agreement(  # noqa: C901
 
         try:
             customer = adobe_client.get_customer(agreement["authorization"]["id"], customer_id)
+            subscriptions = adobe_client.get_subscriptions(
```

Comment:
I would move it outside of `try`. This specific `try`has one single purpose and adding it here make things obscure. Also, AFAIK adobe client methods are wrapped in error handling anyway.